pegasus/sites.v3/code.org/views/carousel_ai_teacher_flexible.haml

```diff
@@ -0,0 +1,24 @@
+- video_ai_teacher_index = 1
+- video_id = ["fireside_chat"]
+- video_ai_teacher_first = ["ik70A1ge-aY"]
+
+.carousel-wrapper#collections_carousel
+  .carousel.carousel--collection
+    .slides
+      - video_ai_teacher_index.times do |index|
+        .slide
+          .action-block.action-block--one-col.flex-space-between
```

Comment:
It's better to use `%figure` or `%div` elements for all of the video wrappers instead of `.action-blocks` since these are just videos, and not cards/action-blocks. 

This will allow you to clean up the related css too.